view.php

```diff
@@ -37,6 +37,8 @@
   exit;
 }
 
+$liiterow["selite"] = str_replace(array("�", "�", "�", "�", "�", "�"), array("a", "a", "o", "A", "A", "O"), $liiterow["selite"]);
```

Comment:
@hannahagberg Kannattanee koodata näin, niin muutkaan ei ASCII-merkit kuin åäö ei riko tota:

    $liiterow["selite"] = preg_replace("/[^a-zA-Z0-9]/", "", $liiterow["selite"]);